internal/connectors/plugins/public/currencycloud/workflow.go

```diff
@@ -8,14 +8,13 @@ func workflow() models.Tasks {
 			TaskType:     models.TASK_FETCH_ACCOUNTS,
 			Name:         "fetch_accounts",
 			Periodically: true,
-			NextTasks: []models.TaskTree{
-				{
-					TaskType:     models.TASK_FETCH_PAYMENTS,
-					Name:         "fetch_payments",
-					Periodically: true,
-					NextTasks:    []models.TaskTree{},
-				},
-			},
+			NextTasks:    []models.TaskTree{},
+		},
+		{
+			TaskType:     models.TASK_FETCH_PAYMENTS,
+			Name:         "fetch_payments",
+			Periodically: true,
+			NextTasks:    []models.TaskTree{},
 		},
```

Comment:
This change is not mentioned by the tickets. What is the reason we moved this from nested to unnested?

Reply:
Yes absolutely !

At first when I developped the currencycloud plugin, I choose to fetch payments per accounts. But it's not specific accounts, we had to fetch the contact id of the account before fetching the payments, which was way too much calls to the PSP if we have to do it by accounts. So I now fetch the payments generally for all accounts and do not need the FromPayload anymore. That's why we can move that to the unested

Reply:
ok makes sense. thanks 🫡